pkg/access/accessPdp.go

```diff
@@ -43,27 +40,22 @@ func checkDissems(dissems []string, entityID string) (bool, error) {
 	return false, nil
 }
 
-func checkAttributes(dataAttrs []Attribute, entitlements []Entitlement, attrDefs []attrs.AttributeDefinition) (bool, error) {
-	zapLog, _ := zap.NewDevelopment()
-
+func checkAttributes(ctx *context.Context, dataAttrs []Attribute, entitlements []Entitlement, attrDefs []attrs.AttributeDefinition) (bool, error) {
```

Comment:
```suggestion
func checkAttributes(ctx context.Context, dataAttrs []Attribute, entitlements []Entitlement, attrDefs []attrs.AttributeDefinition) (bool, error) {
```
`ctx context.Context` is an interface

Reply:
Okay. It looks like I made this mistake in access-pop and may want to fix there as well